frontend/apps/reader/modules/readerhandmade.lua

```diff
@@ -338,8 +342,10 @@ function ReaderHandMade:onDocumentRerendered()
     -- Called with CRE document when partial rerendering not enabled
     self:updateHandmagePages()
     -- Don't have these send events their own events
-    self:setupFlows(true)
-    self:setupToc(true)
+    if Device:isTouchDevice() then
+        self:setupFlows(true)
+        self:setupToc(true)
+    end
     -- ReaderToc will process this event just after us, and will
```

Comment:
Do these 2 self:setupXyz() (but not self:updateHAndmagePages()?) really cause issues when called when nothing has been setup because there was no menu item to do so?
If none, I would just let that be as it was.
(And may be this won't break workflows such as the one I mentionned.)

Reply:
the `setupFlows` definitely caused issues. I can be more specific if you want.

Reply:
fixed now, just for you.